app/[...slug]/page.tsx

```diff
@@ -0,0 +1,55 @@
+import { getStoryblokApi, ISbStoriesParams, StoryblokClient } from '@storyblok/react/rsc';
+import StoryblokStory from '@storyblok/react/story';
+import { resolveRelations } from '@/utilities/resolveRelations';
+
+const activeEnv = process.env.NODE_ENV || 'development';
+
+export default async function Page({ params }) {
+  const { data } = await getStoryData(params);
+
+  return (
+    <StoryblokStory story={data.story} />
+  );
+};
```

Comment:
This `page.tsx` file inside `app/[...slug]` takes care of pages other than the homepage using the dynamic route approach
https://nextjs.org/docs/app/building-your-application/routing/dynamic-routes